app/learning.js

```diff
@@ -7,45 +7,66 @@
  * Main script for initiating the learning app.
  */
 
-define(['jquery', 'bootstrap', 'app/config', 'app/database', 'app/messages', 'app/question', 'app/timer','app/database'], function ($, bootstrap, config, db, messages, question, timer,db) {
-  timer.startTimer(".timer", config.constant("TIME_LIMIT"));
+define(['jquery', 'bootstrap', 'app/config', 'app/database', 'app/messages', 'app/question', 'app/timer','app/database', 'app/ready'], function ($, bootstrap, config, db, messages, question, timer,db,ready) {
+  const THOUSAND = 1000;
```

Comment:
This should explain what's wrong here
http://thedailywtf.com/articles/Avoiding-Magic-Constants